docs/fips.md

```diff
@@ -0,0 +1,38 @@
+# FIPS compliance
+
+The Federal Information Processing Standards (FIPS) are a set of U.S. government standards that ensure the security of computer systems for non-military government agencies and contractors. These standards specify how to perform cryptographic operations, such as encryption, hashing, and digital signatures. FIPS mode is a mode of operation that enforces these standards and rejects any non-compliant algorithms or parameters.
+
+Percona Server for MySQL implements the same level of FIPS support as MySQL. Percona Server for MySQL can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a FIPS-validated version of OpenSSL. You can also receive this functionality by [building Percona Server for MySQL from source code](compile-percona-server.md).
+
+Percona Server for MySQL uses the same variables and values as MySQL. Percona Server for MySQL enables control of FIPS mode on the server side and the client side:
+
+* The `ssl_fips_mode` system variable controls whether the server operates in FIPS mode. This variable is disabled by default.
+
+    The `ssl_fips_mode` system variable has these values:
+
+    * `0` - disables FIPS mode
+    * `1` - enables FIPS mode
+    * `2` - enables `strict` FIPS mode. This value provides more restrictions than the  `1 ` value. However, MySQL itself does not have any FIPS-specific code other than to specify the FIPS mode value to OpenSSL. The exact behavior of FIPS mode for `1` or `2` depends on the OpenSSL version.
+
+* The `--ssl-fips-mode` client option controls whether a given MySQL client operates in FIPS mode. This setting does not change the server setting. This option is disabled by default.
+
+    The `--ssl-fips-mode` client option has these values:
```

Comment:
The same set of values for `--ssl-fips-mode` is used for both server and client command line option.

Reply:
updated